src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Loader } from './Loader';
+import { Person } from '../types';
+import { fetchPeopleData } from '../utils/fetchPeopleData';
+import { generateNavLink } from '../utils/generateNavLink';
+
+export const PeoplePage = () => {
+  const [peopleData, setPeopleData] = React.useState<Person[] | null>(null);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [isError, setError] = React.useState(false);
+
```

Comment:
The variable 'setError' is not consistent with the other state variable names. Consider renaming it to 'setIsError' to maintain consistency.